pypika/queries.py

```diff
@@ -544,7 +544,7 @@ def _where_sql(self, quote_char=None, **kwargs):
 
     def _group_sql(self, quote_char=None, **kwargs):
```

Comment:
You should add `with_alias` to the function signature as a kwarg and ignore the value since if it were to be passed in the kwargs it would cause an exception to be raised when you call `term.get_sql`